logpolicy/logpolicy.go

```diff
@@ -735,3 +738,14 @@ func goVersion() string {
 	}
 	return v
 }
+
+type noopPretendSuccessTransport struct{}
+
+func (noopPretendSuccessTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	io.ReadAll(req.Body)
```

Comment:
IIRC the `RoundTripper` contract is to close the body? From the docs:

> 	// RoundTrip should not modify the request, except for
>	// consuming and closing the Request's Body. RoundTrip may

Reply:
Done. Although in this case it doesn't matter (for now!) because all our Close methods are no-ops. But good for the future.